app/src/Api.tsx

```diff
@@ -195,4 +194,65 @@ export class FirebaseService {
    *    - start with entries: 1, or increment the number of entries
    * 
    */
-}
\ No newline at end of file
+}
+
+export type Task = {
+  title: string;
+  description: string;
+  borough: string;
+  index: number;
+}
+
+export type Prize = {
+  prizeTitle: string;
+  prizeSubtitle: string;
+  description: string;
+  dollarValue: string;
+  ticketValue: number;
+  productLink: string;
+  imageUrl: string;
+}
+
+export class AirTableService {
+  async getTasks(borough?: string): Promise<Task[]> {
+    const rawData = await getAirTableData(TASK_TABLE_NAME);
+    const processedData: Task[] = []
+
+    rawData.forEach((row: any) => {
+      const rowFields = row.fields;
+      const task: Task = {
+        title: rowFields['Task Title'],
+        description: rowFields['Task Description'],
+        borough: rowFields['Borough'],
+        index: rowFields['Index']
+      }
+      if ((borough != undefined && borough === rowFields['Borough']) || borough === undefined) {
```

Comment:
is this saying only add the tasks that are in the requested borough? Maybe add a comment and I'd consider moving up